airbyte-integrations/connectors/destination-oracle-strict-encrypt/metadata.yaml

```diff
@@ -7,17 +7,23 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 3986776d-2319-4de9-8af8-db14c0996e72
-  dockerImageTag: 0.2.0
+  dockerImageTag: 1.0.0
```

Comment:
do we really think this release is worthy of a 1.0.0. label?

Reply:
This is an emotionless `1.0.0` label, it is a backwards incompatible breaking change. If you feel strongly we can bump the minor version, but I don't want to get caught up too much in the ceremony of `1.0.0`

Reply:
yeah, let's keep the 1.0.0 for something worth celebrating

Reply:
heads up @stephane-airbyte  Asked in [slack]([url](https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1712701532113739)) and the "breaking changes" logic in the platform requires a major version bump - going to change this back to `1.0.0`